cocos/3d/skeletal-animation/skeletal-animation-blending.ts

```diff
@@ -354,261 +352,6 @@ export class LegacyBlendStateBuffer extends BlendStateBuffer<LegacyNodeBlendStat
     }
 }
 
-class LayeredVec3PropertyBlendState implements PropertyBlendState<Vec3> {
-    constructor (defaultValue: Readonly<Vec3>) {
-        Vec3.copy(this._defaultValue, defaultValue);
-        Vec3.copy(this.result, defaultValue);
-    }
-
-    public refCount = 0;
-
-    public result = new Vec3();
-
-    public blend (value: Readonly<Vec3>, weight: number): void {
-        this._accumulatedWeight = mixAveragedVec3(
-            this._clipBlendResult,
-            this._clipBlendResult,
-            this._accumulatedWeight,
-            value,
-            weight,
-        );
-    }
-
-    public commitLayerChange (weight: number) {
-        const {
-            result,
-            _clipBlendResult: clipBlendResult,
-            _accumulatedWeight: accumulatedWeight,
-        } = this;
-        if (accumulatedWeight < 1.0) {
-            this.blend(this._defaultValue, 1.0 - accumulatedWeight);
-        }
-        Vec3.lerp(result, result, clipBlendResult, weight);
-        Vec3.zero(this._clipBlendResult);
-        this._accumulatedWeight = 0.0;
-    }
-
-    public reset () {
-        Vec3.copy(this.result, this._defaultValue);
-    }
-
-    private _defaultValue = new Vec3();
-    private _clipBlendResult = new Vec3();
-    private _accumulatedWeight = 0.0;
-}
-
-class LayeredQuatPropertyBlendState implements PropertyBlendState<Quat> {
-    constructor (defaultValue: Readonly<Quat>) {
-        Quat.copy(this._defaultValue, defaultValue);
-        Quat.copy(this.result, defaultValue);
-    }
-
-    public refCount = 0;
-
-    public result = new Quat();
-
-    public blend (value: Readonly<Quat>, weight: number): void {
-        this._accumulatedWeight = mixAveragedQuat(
-            this._clipBlendResult,
-            this._clipBlendResult,
-            this._accumulatedWeight,
-            value,
-            weight,
-        );
-    }
-
-    public commitLayerChange (weight: number) {
-        const {
-            result,
-            _clipBlendResult: clipBlendResult,
-            _accumulatedWeight: accumulatedWeight,
-        } = this;
-        if (accumulatedWeight < 1.0) {
-            this.blend(this._defaultValue, 1.0 - accumulatedWeight);
-        }
-        Quat.slerp(result, result, clipBlendResult, weight);
-        Quat.identity(this._clipBlendResult);
-        this._accumulatedWeight = 0.0;
-    }
-
-    public reset () {
-        Quat.copy(this.result, this._defaultValue);
-    }
-
-    private _defaultValue = new Quat();
-    private _clipBlendResult = new Quat();
-    private _accumulatedWeight = 0.0;
-}
-
-class LayeredNodeBlendState extends NodeBlendState<LayeredVec3PropertyBlendState, LayeredQuatPropertyBlendState> {
-    public setLayerMask (layerIndex: number) {
```

Comment:
Why delete these class? What is the alternative solution

Reply:
It's replaced by the pose stuff.